### PR TITLE
Feature/web 9040 viewing form design updates

### PR DESCRIPTION
**Pull request by Sisawat**
Closes WEB-9040

**Summary**

Updates to ViewingsListForm as per design on WEB-9040

**Change List (describe the changes made to the files)**

- Rearranged fields as per design
- Added optional email fields

**Acceptance Test (how to verify the PR)**

- See that new arrangement of fields matches design/figma
- See that email fields are present and toggleable

**Things to look for during review**

- [ ] Make sure all references to `phillips` class prefix is using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] All strings should be translatable.
- [ ] Unit test should be written and should have a coverage of 90% or higher in all areas.
- [ ] Changes or new components should either write new or update existing documentation.
- [ ] PR should link and close out an existing issue
- [ ] Make sure all commits messages follow convention and are appropriate for the changes

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@phillips/seldon",
-  "version": "1.9.0",
+  "version": "1.9.2",
```
Comment (davidicus) [inline package-lock.json:3]: Weird that it updated to 1.9.2 in your branch. This is at 1.9.3 in main. We may have to try to check this out from main and push up to get rid of this file in your changeset

File: src/components/ViewingsList/ViewingsListCardForm.tsx [hunk 2]
```diff
@@ -295,16 +342,44 @@ const ViewingsListCardForm = ({
...
+        className={classnames(`${baseClass}__preview-set`, { [`${baseClass}__preview-set--hidden`]: !emailOnState })}
+      >
+        <Input
+          id={`email-${id}`}
+          name="email"
+          defaultValue={email}
+          labelText={emailLabel}
+          size="sm"
+          invalid={invalidFields?.address2}
+          invalidText={invalidFields?.address2}
```
Comment (davidicus) [inline src/components/ViewingsList/ViewingsListCardForm.tsx:371]: We may want to apply the hidden attribute based off the `emailOnState` like we do for Preview that you can see on line 303
```js
          hidden={!emailOnState}
```

**Review by davidicus — CHANGES_REQUESTED** [review]
Just the one comment about the package.lock and the comment about the hidden attribute.

File: src/components/ViewingsList/ViewingsListCardForm.tsx (hunk 2 above)
Comment (Sisawat) [reply src/components/ViewingsList/ViewingsListCardForm.tsx:371]: Added.

**Review by davidicus — APPROVED** [review]

**Pull request merged**